platforms/network-platform/settings.gradle

```diff
@@ -0,0 +1 @@
+rootProject.name = 'network-platform'
```

Comment:
Seems there is missing a new line.

Reply:
Newline added by https://github.com/bisq-network/misq/pull/10/commits/4f96b561c1a84fcb0e266b80763a1f6b60bba8a4.